cpp/include/cudf/table/table_view.hpp

```diff
@@ -336,6 +336,22 @@ inline bool has_nested_nulls(table_view const& input)
   });
 }
 
+/**
+ * @brief Returns True if the table has a nullable column at any level of the column hierarchy
+ *
+ * @param input The table to check for nullable columns
+ * @return True if the table has nullable columns at any level of the column hierarchy, false otherwise
+ */
+inline bool has_nested_nullable_columns(table_view const& input)
```

Comment:
Same with this implementation. This could be moved to https://github.com/rapidsai/cudf/blob/branch-23.12/cpp/src/table/table.cpp
This applies to the other non-templated `inline` functions in this header as well so I would be ok if this was done in a follow up PR -- in 24.02 too.

Reply:
Let's make an issue for this.

Reply:
https://github.com/rapidsai/cudf/issues/14431